src/pages/2020.js

```diff
@@ -25,6 +26,11 @@ class Page2020 extends Component {
           image="ogimage-2020.png"
           year="2020"
         />
+        <Helmet>
+          <link rel="preload" as="font" href="https://assets.sparkbox.com/fonts/sole/sole-sans-vf-regular-extra-condensed/font.woff2" type="font/woff2" crossorigin="anonymous" />
+          <link rel="preload" as="font" href="https://assets.sparkbox.com/fonts/solesole-serif-headline-extrabold/font.woff2" type="font/woff2" crossorigin="anonymous" />
+          <link rel="preload" as="font" href="https://assets.sparkbox.com/fonts/solesole-serif-text-light/font.woff2" type="font/woff2" crossorigin="anonymous" />
```

Comment:
Both line 31 and 32 have a typo. They need a / after the first "sole". For example, "https://assets.sparkbox.com/fonts/solesole-serif-headline-extrabold/font.woff2" should be "https://assets.sparkbox.com/fonts/sole/sole-serif-headline-extrabold/font.woff2"